app/src/main/java/org/p2p/wallet/common/analytics/interactor/ScreensAnalyticsInteractor.kt

```diff
@@ -20,6 +21,7 @@ class ScreensAnalyticsInteractor(
         val isNewScreenOpened = newScreenName != previousScreenName
         if (isNewScreenOpened) {
             appScope.launch {
+                Timber.tag("ScreensAnalyticsInteractor",).i("logScreenOpened: $newScreenName")
```

Comment:
You forgot to remove this ?

Reply:
no, i have errors from moonpay at this moment, i want to have additional info on what screen it happens

Reply:
then you have a redundant comma